### ISSUE TITLE
Gif result not well defined

### ISSUE THREAD

**Issue by iib0011** [iib0011/omni-tools#14]
https://omnitools.netlify.app/gif/change-speed

If you use this GIF you will see image wrapped in a black box
![SampleGIFImage_40kbmb](https://github.com/iib0011/omni-tools/assets/58892076/2452d24b-dc7a-44ee-aeef-1918a2bf2ec6)

**Comment by PedroLandolt** [comment]
Hello!

I’m currently pursuing a master’s degree in Software Engineering, and as part of a university project, I need to contribute to an open-source project. I came across your project and found it really interesting—I’d love to work on implementing this feature!

My classmates, lfsjesus and migueljcpedrosa, have already requested their issues as well.

Would you mind assigning this one to me?

Looking forward to contributing. Thank you! 😃

**Comment by iib0011** [comment]
Pls use ffmpeg if possible. You can look at trim video code

**Comment by PedroLandolt** [comment]
Sorry for the delay! Balancing university and work has slowed me down a bit. I’ll start working on this this week and aim to have it finished by the end of the weekend, if not sooner.

**Comment by PedroLandolt** [comment]
![Image](https://github.com/user-attachments/assets/fd230728-26b9-44a5-9bed-2af92d673019)
It's no longer in a black box at least to me, but there is still a yellow shade around the bird for some reason.
Dont know if you had someone fixing this issue or not but still as a bug

**Comment by iib0011** [comment]
Yes I worked on this. Maybe you can debug and try to find why there is a yellow shade

**Comment by PedroLandolt** [comment]
@iib0011 
I fixed the yellow shadow bug but i cant push the content to a branch for some reason.
And i needed that and create a pull request so you could merge so i have prof of helping you on your repo for our class

**Comment by iib0011** [comment]
Next time please create a fork. I just gave you access. You can retry pushing
